From 9c044ecff24fd628d7153e5ccfca2fa984e90acc Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Wed, 1 Jun 2022 16:14:14 +0800
Subject: [PATCH 15/31] Revert "drm/amdkfd: Fix -Wstrict-prototypes from
 amdgpu_amdkfd_gfx_10_0_get_functions()"

This reverts commit 33fcb359a64213bea0510770abc761de9780ca89.
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gfx_v10.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gfx_v10.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gfx_v10.c
index f7c4337c1ffe..ce30d4e8bf25 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gfx_v10.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gfx_v10.c
@@ -165,7 +165,7 @@ static const struct kfd2kgd_calls kfd2kgd = {
 	.get_tile_config = amdgpu_amdkfd_get_tile_config,
 };
 
-struct kfd2kgd_calls *amdgpu_amdkfd_gfx_10_0_get_functions(void)
+struct kfd2kgd_calls *amdgpu_amdkfd_gfx_10_0_get_functions()
 {
 	return (struct kfd2kgd_calls *)&kfd2kgd;
 }
-- 
2.27.0

